cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -159,6 +176,11 @@ if ( deprecatedConfigFile.exists() && configFile.exists() ) {
 private def artifactCleanup(String timeUnit, int timeInterval, String[] repos, log, paceTimeMS, dryRun = false, disablePropertiesSupport = false) {
     log.info "Starting artifact cleanup for repositories $repos, until $timeInterval ${timeUnit}s ago with pacing interval $paceTimeMS ms, dryrun: $dryRun, disablePropertiesSupport: $disablePropertiesSupport"
 
+    if ( !repos || (repos.length == 0) ) {
```

Comment:
All these checks are identical except for the message. Can we DRY this up?

Reply:
Oh they are different use cases and I am trying to minimize impacts

Reply:
Different use cases? I don't understand. They are used in different place of the code, sure but the code is identical.

Reply:
i had it in only 1 place inside artifactCleanup() but that would execute all extra stuff and fail at last minute so even though repeating the message clarifies where it gets hit

Reply:
So your point is to wrap into a function and call each place with a different message ?

Reply:
I'm still confused. Perhaps it's my lack of Groovy knowledge? I fail to see how a private function containing these code, and being called from all these locations won't work?

Reply:
> So your point is to wrap into a function and call each place with a different message ?

Yes, of course. How else would one DRY these code up?

Reply:
DRY = Don't Repeat Yourself, or don't create duplication.

Reply:
Not used to that acronym!!! But it is done.. Will reverify locally and ask support person to confirm  in customer reproduction as well.

Reply:
> if it is a function you want to call with a different message I can do that

Yes, that's what I meant.